source/manual/manually-resize-ebs.html.md

```diff
@@ -0,0 +1,102 @@
+---
+owner_slack: "#re-govuk"
+title: Manually resize EBSs (elastic block store)
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-12-01
+---
+
+In AWS, [we use Elastic Block Store (EBS)][ebs] to attach additional
+block storage to EC2 instances.
+
+EBS is not the same thing as [Elastic File System (EFS)][efs], which
+is more like a shared NFS volume and does not need to be manually
+resized.
+
+[ebs]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html
+[efs]: https://docs.aws.amazon.com/efs/latest/ug/whatisefs.html
+[brexit-tool]: https://www.gov.uk/get-ready-brexit-check
```

Comment:
I don't see this used anywhere
```suggestion
```

Reply:
The risks of copying pre-existing pages...